src/test/java/com/datadog/api/TestUtils.java

```diff
@@ -139,6 +143,10 @@ public HttpURLConnection getConnection(URL url) throws IOException {
         protected static String TEST_API_KEY;
         protected static String TEST_APP_KEY;
         protected static int WIREMOCK_PORT = 8080 + SUREFIRE_FORK;
+        // We need to make the tag be something that is then searchable in monitors
+        // https://docs.datadoghq.com/tracing/guide/metrics_namespace/#errors
+        // "version" is really the only one we can use here
+        protected static final String TRACING_TAG_ENDPOINT = "version";
```

Comment:
version should be the version of the library

Reply:
See the comment above.